spec/acceptance/basic_webhook_spec.rb

```diff
@@ -1,9 +1,8 @@
 require 'spec_helper_acceptance'
-require 'json'
+
 describe 'System Ruby with No SSL, Not protected, No mcollective' do
   context 'default parameters' do
-    let(:pp) do
-      "
+    pp = %(
```

Comment:
Why should this not be a let block anymore?

Reply:
IMO, it is clearer in this way.

Reply:
There is a difference in evaluation and https://stackoverflow.com/questions/5359558/when-to-use-rspec-let/5359979#5359979 outlines why let blocks are preferred.